zmon_aws_agent/main.py

```diff
@@ -147,6 +148,7 @@ def main():
         dynamodbs = aws.get_dynamodb_tables(region, infrastructure_account)
         certificates = aws.get_certificates(region, infrastructure_account)
         aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs)
+        sqs = aws.get_sqs_queues(region, infrastructure_account)
```

Comment:
Since we've had some rate limiting issues before... Do you think polling for SQS resources could use a longer interval?
At least until we have more information on #94